src/OrchardCore/OrchardCore.ReCaptcha.Core/Services/ReCaptchaService.cs

```diff
@@ -124,8 +125,10 @@ private async Task<bool> VerifyAsync(string responseToken)
                     { "response", responseToken }
                 });
 
-                _httpClient ??= _httpClientFactory.CreateClient(nameof(ReCaptchaService));
-                var response = await _httpClient.PostAsync($"{_reCaptchaSettings.ReCaptchaApiUri.TrimEnd('/')}/siteverify", content);
+                var httpClient = _httpClientFactory.CreateClient(nameof(ReCaptchaService));
+                _verifyHost ??= $"{_reCaptchaSettings.ReCaptchaApiUri.TrimEnd('/')}/siteverify";
```

Comment:
move to the constructor, make field readonly

Reply:
Yes

Reply:
@sebastienros done. Is there a reason for this request? what I had did the same thing except that we don't initialize until needed.

Reply:
it doesn't need to be tested for null each time.

Reply:
okay. let me know if you want anything else changed or let's merge it